Sources/MySQL/Database.swift

```diff
@@ -74,6 +78,7 @@ public final class Database {
     private let socket: String?
     private let flag: UInt
     private let encoding: String
+    public static var fractionalSecondsDigits: Int = 0
```

Comment:
Why should this variable be `static`? Maybe not all database connections share the same settings.